Lesson7/feyza/Lesson4-insight/styles.css

```
@@ -0,0 +1,95 @@
+@import url('https://fonts.googleapis.com/css2?family=Nunito&display=swap');
+
+* {
+  box-sizing: border-box;
+}
+
+body {
+  background: linear-gradient(90deg, rgba(2, 0, 36, 1) 0%, rgba(8, 8, 110, 1) 0%, rgba(5, 4, 75, 1) 0%, rgba(41, 128, 219, 1) 0%, rgba(153, 0, 255, 1) 65%);
+  margin: 0;
+  font-family: "Nunito", sans-serif;
+  padding: 0;
+}
+
+header {
+  background-color: #fff;
+  width: 100%;
+  height: 80px;
+  display: inline-block;
+}
+
+.navbar {
+  width: 100%;
+  text-align: center;
+
+}
+
+.navbar ul {
+  list-style-type: none;
+  display: flex;
+  justify-content: space-around;
+  align-items: center;
+
```

Add some space between menu items
```suggestion
  gap: 20px;
```